src/e2e-test/features/dataplex/source/RunTimeDataplex.feature

```diff
@@ -0,0 +1,136 @@
+
+  @BATCH-TS-DATAPLEX-RunTime-02
+  Scenario Outline: To verify the pipelines is successfully deployed and run for different format when asset type is
+  storage bucket
+    When Open Datafusion Project to configure pipeline
+    Then Select plugin: "Dataplex" from the plugins list as: "Source"
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Connect plugins: "Dataplex" and "Dataplex2" to establish connection
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexEntityID" with value: "dataplex.entity"
+    And Click on the Get Schema button
+    And Validate "Dataplex" plugin properties
+    And Close the Plugin Properties page
+    And Navigate to the properties page of plugin: "Dataplex2"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetStorageBucket"
+    And Select radio button plugin property: "dataplexAssetType" with value: "dataplex.storageBucketRadioBtn"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.outputTable"
+    And Select dropdown plugin property: "dataplexFormat" with option value: "<Format>"
+    And Validate "Dataplex" plugin properties
+    And Close the Plugin Properties page
+    And Save the pipeline
+    And Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+    Then Validate OUT record count is equal to IN record count
+    Examples:
+      | Format  |
+      | avro    |
+      | json    |
+      | parquet |
+
+  @BATCH-TS-DATAPLEX-RunTime-03
+  Scenario Outline: To verify the orc format pipeline is successfully deployed when asset type is storage bucket
```

Comment:
why do we have a separate test for orc, csv?

Reply:
To test orc, csv formats we've to use different prerequisite data which support datatypes accepted by orc and csv.

Reply:
Please add a comment explaining why orc and csv are handled separately (also mention why you need different data for these formats). Also the Scenario Outline should be "To verify the orc and csv formats ..."

Reply:
Comment added in the test case scenario. 
Also scenario outline description changed.